### DISCUSSION TITLE
Is there a way to tell branchless that a commit has not been rewritten?

### DISCUSSION THREAD

**Discussion by pokey** [arxanas/git-branchless#630] [Q&A] [answered]
I have somehow ended up in a situation where branchless thinks that my `main` branch has been rewritten as another commit.  I have no idea how that happened, but it would be good if I could inform branchless that this is not the case, so it stops suggesting that I try to restack `main`.  Here's the relevant portion of my smartlog:

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/755842/200531816-34a5c39f-8d78-4769-97a6-edc2ca99045c.png">

**Comment by arxanas** [comment, marked as answer]
It's a bit of a hack, but  does `git unhide main` work to fix it?

**Reply by pokey** [reply]
Yep that did it thanks!

**Reply by akheron** [reply]
I just ran to this issue today, and this comment helped me fix it. I wonder if there would be an obvious place in the wiki where to add this info for better discoverability? 🤔
